setup.py

```diff
@@ -14,7 +14,7 @@
     long_description=long_description,
     long_description_content_type="text/markdown",
     url="https://github.com/richardingham/octopus",
-    packages=["octopus", "octopus.manufacturer", "octopus.blocks"],
+    packages=["octopus", "octopus.manufacturer", "octopus.blocktopus.blocks"],
```

Comment:
I think this is fixed by 53a9188 but if you could test on your system that would be great. 
octopus.blocks is a new namespace package for plugins. octopus.blocktopus.blocks is the existing blocks.

Reply:
Indeed,fixed by https://github.com/richardingham/octopus/commit/53a91880606b3551ef3c7ac7b944c07c84807f83, rolled back now